library/core/src/num/uint_macros.rs

```diff
@@ -1017,6 +1017,44 @@ macro_rules! uint_impl {
             }
         }
 
+        /// Unchecked integer division. Computes `self / rhs`.
+        ///
+        /// Calling `x.unchecked_div(y)` is semantically equivalent to calling
+        /// `x.`[`checked_div`]`(y).`[`unwrap_unchecked`]`()`.
+        ///
+        /// If you're just trying to avoid the panic in debug mode, then **do not**
+        /// use this.  Instead, you're looking for [`wrapping_div`].
+        ///
+        /// # Safety
+        ///
+        /// This results in undefined behavior when
+        #[doc = concat!("`rhs == 0`")]
+        /// i.e. when [`checked_div`] would return `None`.
+        ///
+        /// [`unwrap_unchecked`]: option/enum.Option.html#method.unwrap_unchecked
```

Comment:
```suggestion
        /// [`unwrap_unchecked`]: Option::unwrap_unchecked
```
using paths in intradoc links is preferred as it is generally more compatible with tools like rust-analyzer and rustrover

Reply:
Rather odd, faced an issue with rustdoc when I made this change, but passes when I revert.

Reply:
odd but not a big deal since it still works